Week 4/GroceryStore.html

```diff
@@ -9,8 +9,8 @@
 	<p></p>
 	<script>
 		$("button").click(() => {
-      		const ItemName = $("input#grocery").val();
-      		if (ItemName = "Eggs") {
+      		const itemName = $("input#grocery").val();
+      		if (itemName = "Eggs") {
```

Comment:
The reason your code is erroring is because the variable is being assigned to "eggs". Notice how you are using a single equal sign instead of a double which is normally used for comparison.